.typos.toml

```diff
@@ -0,0 +1,32 @@
+[files]
+extend-exclude = [
+    ".git/",
+    ".phpunit.result.cache",
+    "flags/??.png",
+    "node_modules/",
+    "settings/flags.php",
+    "settings/languages.php",
+    "tmp/",
+    "vendor/",
+	".typos.toml",
```

Comment:
The config file is always excluded.
```suggestion
```

Reply:
That's true, but the Typos VS Code extension doesn't. Cf this [discussion](https://github.com/polylang/polylang/pull/1433#discussion_r1514721022).

Reply:
Reported in https://github.com/tekumara/typos-lsp/issues/47

Reply:
Thanks @szepeviktor!

Reply:
@Hug0-Drelon Released in https://github.com/tekumara/typos-lsp/releases/tag/v0.1.16